### ISSUE TITLE
Refinements to the Audit Start page

### ISSUE THREAD

**Issue by melchoyce** [GSA-TTS/FAC#375]
Breakout of #264.

### Background

@sreuber and @austinhernandez have done a lot of great work on the Audit flow! Let's make sure final refinements are reflected across the whole Figma doc.

Can we identify which is the "final" design and then make sure all iterations of the screen in Figma match?

This screen appears in:
- #241 Auditee information - invalid UEI
- #260 Report home CTAs / audit start page CTAs
- #264 Report home / audit start page
- #321 - Certify

**Comment by melchoyce** [comment]
Looks like this is good to go, @austinhernandez can you confirm?

**Comment by austinhernandez** [comment]
not quite...... i think this ticket will be complete when there is a completely updated, consistent user flow ready for the next round of usability testing

**Comment by austinhernandez** [comment]
This is a sprint 11 prototype placeholder where we can put updated / consistent screens with finalized copy ready for usability testing: 

https://www.figma.com/file/8REinLFKSMj7pwQ7JIEHGW/FAC-Sprint-11-prototype?node-id=0%3A1

**Comment by austinhernandez** [comment]
This is what we are calling the `Audit start page`, also sometimes referred to as the `report start page`. I am declaring this "done." Done meaning it is ready for usability testing with Auditees (#460)

[Figma link to this page in the prototype here](https://www.figma.com/file/8REinLFKSMj7pwQ7JIEHGW/FAC-Sprint-11-prototype?node-id=599%3A2351)

![image](https://user-images.githubusercontent.com/23194720/190421889-07ec725e-ee30-4258-91c2-9a15ff6d17ae.png)

Notes:
The only thing that _might_ change is the report meta data at the bottom which is being worked on in https://github.com/GSA-TTS/FAC/issues/445
![image](https://user-images.githubusercontent.com/23194720/190423428-36d7491a-1191-46a8-9637-0c9fa86b8d73.png)

**Comment by petermarks-gsa** [comment]
@austinhernandez A [previous design](https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=3136%3A18722) we scoped https://github.com/GSA-TTS/FAC/issues/212 against outlines seven alternate states of this screen. Is it possible to get the seven alternate states of this screen with the updated design or update the previous design?

**Comment by geekygirlsarah** [comment]
Is this safe to close now? Anything need to happen before we do?

**Comment by melchoyce** [comment]
Safe to close, and being superseded by https://github.com/GSA-TTS/FAC/issues/536.
